scripts/devops_tasks/build_conda_artifacts.py

```diff
@@ -0,0 +1,302 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# Used to generate conda artifacts given a properly formatted build folder
+#
+#   EXAMPLE: examine the CondaArtifacts in /sdk/storage/meta.yaml
+#
+#   Grab the source code from each of the tags in the CondaArtifact.
+#
+#   Format the directory that you pass to the "build_directory" argument in this way
+#   <build directory>
+#       /azure-storage-blob <-- package folder from tag specified
+#           /setup.py
+#           /...
+#       /azure-storage-queue
+#       /azure-storage-file-datalake
+#       /azure-storage-fileshare
+
+import argparse
+import sys
+import os
+import shutil
+import re
+
+from common_tasks import process_glob_string, run_check_call, str_to_bool, parse_setup
+from subprocess import check_call
+from distutils.dir_util import copy_tree
+
+VERSION_REGEX = re.compile(r"\{\%\s*set\s*version\s*=\s*\"(.*)\"\s*\%\}")
+
+NAMESPACE_EXTENSION_TEMPLATE = """__path__ = __import__('pkgutil').extend_path(__path__, __name__)  # type: str
+"""
+
+MANIFEST_TEMPLATE = """include *.md
+{namespace_includes}
+include LICENSE.txt
+recursive-include tests *.py
+recursive-include samples *.py *.md
+"""
```

Comment:
I've noticed this inconsistency, is it practice to include LICENSE.txt? Some libraries have it included and others do not. It's not in the [template](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/template/azure-template/MANIFEST.in) but [tables](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/tables/azure-data-tables/MANIFEST.in) does include it.

Reply:
Thanks. will do it.

Reply:
We should PR this + the long_description into template too.